lib/active_model/serializer.rb

```diff
@@ -122,8 +122,12 @@ def initialize(object, options = {})
       end
     end
 
+    def object_model_name
+      object.class.model_name
+    end
+
```

Comment:
@bf4 IMO I don't think the serializer should have any methods on it relating to a specific adapter.